src/lib.rs

```diff
@@ -107,7 +108,7 @@ pub use bundle::DynamicClone;
 #[doc(hidden)]
 pub use query::Fetch;
 
-#[cfg(feature = "macros")]
+#[cfg(any(feature = "macros", feature = "macros_no_std"))]
```

Comment:
Is the second feature flag here a left-over of some kind? I don't see where it is introduced.

Reply:
Yes this is a left-over. Removed.